.github/workflows/linux_android_arm64.yml

```diff
@@ -0,0 +1,178 @@
+          
+          # vcpkg tree of dependencies require extra packages
+          sudo -E apt --assume-yes install pkg-config linux-libc-dev
+          
+          # Install Android SDK, NDK and Tools
+          sudo apt -y --no-install-recommends install unzip
+          wget https://dl.google.com/android/repository/commandlinetools-linux-7583922_latest.zip
+          unzip commandlinetools-linux-7583922_latest.zip
+          echo "yes" | ./cmdline-tools/bin/sdkmanager --sdk_root=${{ env.ANDROID_TOOLS }} --install "ndk-bundle" "platform-tools" "platforms;android-${{ env.ANDROID_SDK_VERSION }}"
+
+      - name: Setup ccache
+        uses: hendrikmuhs/ccache-action@v1.2
+        with:
+          max-size: "2000M"
+          # Should save cache only if run in the master branch of the base repo
+          # github.ref_name is 'ref/PR_#' in case of the PR, and 'branch_name' when executed on push
+          save: ${{ github.ref_name == 'master' && 'true' || 'false'  }}
+          verbose: 2
+          key: ${{ github.job }}-linux-android-arm64
+          restore-keys: |
+            ${{ github.job }}-linux-android-arm64
+
+      - name: Get tools versions
+        run: |
+          ninja --version
+          ccache --version
+          python3 --version
+          cmake --version
+
+      #
+      # Build
+      #
+
+      - name: Build vcpkg
+        run: |
+          ${{ env.VCPKG_ROOT }}/bootstrap-vcpkg.sh --disableMetrics
+          # patch vcpkg default (community) toolchain to build only Release configuration
+          echo "set(VCPKG_BUILD_TYPE release)" >> ${{ env.VCPKG_ROOT }}/triplets/community/arm64-android.cmake
+
+      - name: Get number of CPU cores
+        uses: SimenB/github-actions-cpu-cores@v1
+        id: cpu-cores
+
+      - name: CMake configure
+        run: |
+          cmake \
+            -G Ninja \
+            -DCMAKE_VERBOSE_MAKEFILE=ON \
+            -DCMAKE_BUILD_TYPE=${{ env.BUILD_TYPE }} \
+            -DVCPKG_TARGET_TRIPLET=arm64-android \
+            -DVCPKG_HOST_TRIPLET=x64-linux-release \
+            -DCMAKE_TOOLCHAIN_FILE=${{ env.VCPKG_ROOT }}/scripts/buildsystems/vcpkg.cmake \
+            -DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=${{ env.ANDROID_NDK_HOME }}/build/cmake/android.toolchain.cmake \
+            -DCMAKE_COMPILE_WARNING_AS_ERROR=ON \
+            -DANDROID_ABI=${{ env.ANDROID_ABI_CONFIG }} \
+            -DANDROID_PLATFORM=${{ env.ANDROID_SDK_VERSION }} \
+            -DENABLE_PYTHON=OFF \
+            -DENABLE_SYSTEM_OPENCL=ON \
+            -DENABLE_SYSTEM_PROTOBUF=ON \
+            -DENABLE_SYSTEM_PUGIXML=ON \
+            -DENABLE_SYSTEM_SNAPPY=ON \
+            -DENABLE_SYSTEM_TBB=ON \
+            -DENABLE_SYSTEM_FLATBUFFERS=ON \
+            -DENABLE_INTEL_GPU=ON \
+            -DENABLE_TESTS=ON \
+            -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
+            -DCMAKE_C_COMPILER_LAUNCHER=ccache \
+            -DCMAKE_CXX_LINKER_LAUNCHER=ccache \
+            -DCMAKE_C_LINKER_LAUNCHER=ccache \
```

Comment:
let's remove LINKER cache in every pipeline to minimize cache size (also, it's low probable that binaries are reused as is from cache)